marketplace/commercetools-products/src/apollo.js

```diff
@@ -0,0 +1,41 @@
+import SdkAuth, { TokenProvider } from '@commercetools/sdk-auth';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { createHttpLink } from 'apollo-link-http';
+import { setContext } from 'apollo-link-context';
+
+export async function getApolloClient({ authUri, projectKey, clientId, apiUri, clientSecret }) {
+  // @todo how do I get the refreshed token?
```

Comment:
Can you elaborate? Is it possible for a long session to expire a token? Should we poll for its validity?